R/misc_yhat.R

```diff
@@ -90,6 +95,10 @@ yhat.cv.glmnet <- function(X.model, newdata, ...) {
   }
   if (!is.null(X.model$glmnet.fit$classnames)) {
     pred <- predict(X.model, newdata, type = "response", s = X.model$lambda[length(X.model$lambda)])
+    #glmnet returns and 3D array for multilabel classif
+    if(!is.null(dim(pred)[3])){
```

Comment:
same as for `yhat.gbm`
btw, what is in the first and second dimension of these methods?

Reply:
Done